README.md

```diff
@@ -1,8 +1,8 @@
 # Sui README
 
-![Sui icon](doc/static/Sui_Icon_Brand.png)
-
-Welcome to Sui, a next generation smart contract platform with high throughput, low latency, and an asset-oriented programming model powered by the [Move](https://github.com/MystenLabs/awesome-move) programming language! Find everything you need within the [Sui Developer Guides](doc/src/learn/index.md) and supporting materials.
+|             |             |
+| ----------- | ----------- |
+| Welcome to Sui, a next-generation smart contract platform with high throughput, low latency, and an asset-oriented programming model powered by the [Move programming language](https://github.com/MystenLabs/awesome-move)! Find everything you need within the [Sui Developer Guides](doc/src/learn/index.md) and supporting materials below.      | ![Sui icon](doc/static/Sui_Icon_Brand.png)       |
```

Comment:
- The icon still seems a bit big--maybe we should embed it in HTML + resize? e.g., `<img src="doc/static/Sui_Icon_Brand.png" alt="sui_icon" width="200"/>`
- There's a lot of extra whitespace that hides the good README content that comes after the beginning. I'm not sure if that comes from the table or whitespace in the icon itself, but would be good to eliminate/minimize.

Reply:
Heh, I went with HTML but converted to Markdown because I know Francois prefers it. Let me experiment.